lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -220,12 +216,6 @@ export class RedirectClient extends StandardInteractionClient {
                 this.browserStorage.cleanRequestByInteractionType(
                     InteractionType.Redirect
                 );
-                if (performanceClient && correlationId) {
-                    performanceClient?.addFields(
-                        { errorCode: "no_server_response" },
```

Comment:
This was actually the correct place to put this because here we know we were expecting a response but didn't get one.